MainWindow.xaml.cs

```diff
@@ -21,12 +21,15 @@ public partial class MainWindow : Window
         private MediaType? _currentMediaType { get; set; }
         private List<int> _driveSpeeds { get { return new List<int> { 1, 2, 3, 4, 6, 8, 12, 16, 20, 24, 32, 40, 44, 48, 52, 56, 72 }; } }
         private List<KeyValuePair<string, KnownSystem?>> _systems { get; set; }
-        private List<KeyValuePair<string, MediaType?>> _mediaTypes { get; set; }
-        private Process childProcess { get; set; }
+        private List<MediaType?> _mediaTypes { get; set; }
+        //private Process childProcess { get; set; }
 
         private OptionsWindow _optionsWindow;
         private Options _options;
 
+        private DumpEnvironment _env;
+
+
```

Comment:
Nit: Since we have a lot of locals now, might be worth reordering them and grouping them for clarification.
Secondary even more minor nit: generally don't have double lines between stuff. If grouping is needed, we can use the `#region` and `#endregion` tags